### DISCUSSION TITLE
Any plans to upgrade to rewrite v8 ?

### DISCUSSION THREAD

**Discussion by shresthaujjwal** [spring-projects-experimental/spring-boot-migrator#845] [Q&A] [answered]
Any plans to upgrade to openrewrite to v8 as it supports gradle ? 
Between great migrator app love it.

**Comment by fabapp2** [comment, marked as answer]
Hi @shresthaujjwal 
Thanks for your interest in Spring Boot Migrator 🚀
Yes, we are currently working in a reimplementation from the parsing up to recipes. 
The new module (currently) 'sbm-support-rewrite' will use latest OR and support Maven, Gradle and Java parsing from OpenRewrite (need to see what it takes to support Kotlin too).
I will provide more information about the path forward soon.

**Reply by shresthaujjwal** [reply]
thanks @fabapp2 is this the EPIC that track the gradle support ? https://github.com/spring-projects-experimental/spring-boot-migrator/issues/844
